examples/parse_folder.rs

```diff
@@ -0,0 +1,70 @@
+/// This an example usage of the rustpython_parser crate.
+/// This program crawls over a directory of python files and
+/// tries to parse them into an abstract syntax tree (AST)
+///
+/// example usage:
+/// $ RUST_LOG=info cargo run --release parse_folder /usr/lib/python3.7
+
+#[macro_use]
+extern crate clap;
+extern crate env_logger;
+#[macro_use]
+extern crate log;
+
+use clap::{App, Arg};
+
+use rustpython_parser::{ast, parser};
+use std::path::Path;
+
+fn main() {
+    env_logger::init();
+    let app = App::new("RustPython")
```

Comment:
Since `RustPython` is the name of the whole project and, presumably, its interpreter as a stand-alone application – shouldn’t this be called something else (or at least – more specific)?

Reply:
Oops, copy paste error, thanks!